.eslintrc.json

```diff
@@ -9,35 +9,55 @@
           "error",
           {
             "enforceBuildableLibDependency": true,
-            "allow": ["regenerator-runtime/runtime"],
+            "allow": [
+              "regenerator-runtime/runtime",
+              "@angular/router",
```

Comment:
<insert me angry over nx linting errors that shouldnt happen>

Reply:
??

Reply:
i dont remember but its just something with nx thats causing lint errors here that shouldnt happen